src/components/ClaimChildPanel.js

```diff
@@ -330,7 +329,7 @@ class ClaimChildPanel extends Component {
       itemFormatters.push(
         (i, idx) => (
           <PublishedComponent
-            readOnly={true}
+            readOnly={!i.product?.uuid}
```

Comment:
I think this is not enough, while it will provide check required from the product POV, sometimes rejecting/approving should be blocked - @delcroip @dragos-dobre shouldn't it?

Reply:
I actually think it will cover all cases because if the item/service is rejected by the check I think the prod is empty